MonstercatNet/MonstercatApi.cs

```diff
@@ -221,5 +222,25 @@ public Task<HttpContent> StreamTrack(TrackStreamRequest request)
 
             return _service.StreamTrack(request);
         }
+
+        public Task CreatePlaylist(Playlist request)
```

Comment:
This endpoint returns the created playlist. In order to be able to delete the playlist, we need to return that. Can you take a look at this again?

Reply:
As explained in the comment above.